llvm_util/llvm2alive.cpp

```diff
@@ -596,11 +614,9 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
       }
     }
 
-    // infinite void functions may not have a return stmt in LLVM
+    // a function with infinite loop may not have a return stmt
     if (!Fn.hasReturn()) {
-      assert(f.getReturnType()->isVoidTy());
-      Fn.getBBs().back()->addInstr(
-        make_unique<Return>(Type::voidTy, Value::voidVal));
```

Comment:
This assertion is removed because non-void function can have an infinite loop as well:

```
; This is valid 
define i32 @f() {
br label %loop
loop:
br label %loop
}
```